pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.wavefront.tools</groupId>
     <artifactId>wftop</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.3-SNAPSHOT</version>
```

Comment:
can we make this also to 1.2, so all will be in sync

Reply:
will the version not point to the upcoming release i.e `1.3`? because of which I've mentioned `1.3-SNAPSHOT` so that whenever we build after 1.2 release, it'll be a snapshot for 1.3.

Reply:
Whenever we make new changes at that time we can make this to 1.3, and tag the current version to 1.2 for now 1.2 looks fine to me

Reply:
Changed the version to `1.2`